### DISCUSSION TITLE
路遇格陵兰睡鲨，我需要赶紧跑吗？急！！！！！！！！

### DISCUSSION THREAD

**Discussion by XokoukiguraX** [share121/inter-knot#935] [其他]
![微信图片_20240720194623](https://github.com/user-attachments/assets/ac718296-37b6-4c0b-9395-458ad96a6f1e)

![微信图片_20240720194335](https://github.com/user-attachments/assets/ba4f8149-a667-4df3-8bd9-1f5d9dafea55)

**Comment by warning-Kirov-reporting** [comment]
好机会，偷偷摸摸尾巴！

**Reply by XokoukiguraX** [reply]
这，不会被咬吧

**Comment by XokoukiguraX** [comment]
这，不会被咬吧

**Comment by kaijinti** [comment]
楼主还活着吗？

**Comment by gaojiaobei** [comment]
这个妹妹我认得的，是维多利亚家政的，当时去六分街录像店的时候见过，是合法公民，鲨希人，据说还是学生，不会咬人的。
不过话说她成年了吗？维多利亚家政不会雇佣童工吧

**Comment by XokoukiguraX** [comment]
有新艾利都医院的吗，刚刚摸了一下，她不知道从哪里拿出来把剪刀，把我右手剪到了，现在流的血都结霜了（幸好我跑得快，捡回来一条命）

**Comment by XokoukiguraX** [comment]
怎么回事，我怎么在路上一直大喊‘耶比——’？搞的两个路人过来非要嘉着我去精神病院

**Comment by Larksonguesinaspic** [comment]
楼主大约的确是去ICU了

**Comment by Qiling-XwX** [comment]
楼主可以把这个妹妹抱到我家嘛（）

**Comment by GanlinSama** [comment]
祝楼主早日康复🙏
